spec/features/peep_spec.rb

```diff
@@ -0,0 +1,13 @@
+require_relative './web_helpers.rb'
+
+feature 'create a peep' do
+  scenario 'a user can create a peep' do
```

Comment:
Another test you might consider adding is testing that not logged in users can't create peeps.